### PR TITLE
Enable UART1 hardware flow control on frdm-K64f board

### PR DESCRIPTION
**Pull request by devran01**
This PR enables Serial Flow Control and adds pin map for UART1 RTS and CTS pins.

### CODE REVIEW

**Comment by devran01** [comment]
@0xc0170 
Please review and merge if appropriate.

File: mbed-hal-frdm-k64f/device.h [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 #define DEVICE_SERIAL           1
 #define DEVICE_SERIAL_ASYNCH    1
+#define DEVICE_SERIAL_FC		1
```
Comment (0xc0170) [inline mbed-hal-frdm-k64f/device.h]: Please fix the formatting. I assume tabs? same for hte other files

**Comment by devran01** [comment]
@0xc0170 fixed the formatting issues. Please merge.

**Comment by 0xc0170** [comment]
LGTM

**Pull request merged**